website/client/components/inventory/equipment.vue

```diff
@@ -65,12 +65,18 @@
           v-for="(label, group) in gearTypesToStrings",
           :key="group",
           :item="flatGear[activeItems[group]]",
+          :showPopover="flatGear[activeItems[group]] && flatGear[activeItems[group]].key.indexOf('_base_0') === -1",
           :label="$t(label)",
           :selected="true",
           :popoverPosition="'top'",
           :starVisible="!costume || user.preferences.costume",
           @click="equip",
         )
+          template(slot="itemContent", scope="ctx")
+            div(:class="'shop_' + ctx.item.key")
```

Comment:
What if instead of a slot the item content was just a prop passed to the item component since the only thing that changes is the class? But maybe a full slot gives more flexibility